apps/betterangels-backend/clients/models.py

```diff
@@ -96,7 +96,7 @@ class Meta:
     pghistory.DeleteEvent("client_profile.remove"),
 )
 class ClientProfile(BaseModel):
-    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="client_profile")
+    user = models.OneToOneField(User, on_delete=models.SET_NULL, blank=True, null=True, related_name="client_profile")
```

Comment:
Makes sense, this is optional so we can associate users if we want in the future.

Do we have a ticket to delete client user accounts?